packages/shell-chrome/views/_partials/data/attribute.edge

```diff
@@ -0,0 +1,138 @@
+<div class="group flex items-center">
+    <a
+        :style="\`margin-left: \${singleData.depth}px; \`"
```

Comment:
Guess this is the only downside of Edge.js 😄

Reply:
Eyes like a hawk! Yeah...unfortunately so 🙄 Will have a think about a way around this if poss.

Reply:
We could move it into the JS file, maybe even use `x-spread`

Reply:
Yeah, I mean we could just do `'margin-left: ' +  singleData.depth + 'px'` but it'd be nice to have the option to use backticks without escaping them eh.

Reply:
So we can use a custom tag to wrap the block here and escape the backticks/dollar. I've tested this:

    @escape
        <div :title="isLatest? 'Latest Version' : `Latest Version: ${latest}`"></div>
    @endescape

Generates:

    <div :title="isLatest? 'Latest Version' : \`Latest Version: \${latest}\`"></div>

Any thoughts on this? Probably needs less generic name.

Reply:
It's fine, let's leave the escaped characters in for now, we can find a better solution if it becomes painful